models/models_methods.go

```diff
@@ -779,7 +779,7 @@ func MergeStepWith(step, otherStep stepmanModels.StepModel) (stepmanModels.StepM
 // --- WorkflowIDData
 
 // GetWorkflowIDFromListItemModel ...
-func GetWorkflowIDFromListItemModel(workflowListItem WorkflowListItemModel) (string, error) {
+func GetWorkflowIDFromListItemModel(workflowListItem StageWorkflowListItemModel) (string, error) {
```

Comment:
NIT: shouldn't we rename the function to reflect better that this is working with stage workflow list models? Something like: `GetWorkflowIDFromStageWorkflowListItemModel` ?

Also, it seems this is only used to validate the stages (`validateStages`), the validation func is private, do we need this to be a public func?

Reply:
Good idea. I made them private and also shortened their name because it did not seem necessary to include the type name in the functions name.